cmd/step-ca/main.go

```diff
@@ -141,7 +142,21 @@ Run the Step CA and read the password from a file - this is useful for
 automating deployment:
 '''
 $ step-ca $STEPPATH/config/ca.json --password-file ./password.txt
-'''`
+'''
+Run the Step CA for the context named _mybiz_ and prompt for password:
+'''
+$ step-ca --context=mybiz
+'''
+Run the Step CA for the context named _mybiz_ and an alternate ca.json file:
+'''
+$ step-ca --context=mybiz other-ca.json
+'''
+Run the Step CA for the context named _mybiz_ and read the password from a file - this is useful for
+automating deployment:
+'''
+$ step-ca --context=mybiz --password-file ./password.txt
+'''
```

Comment:
We should add an example for running just `step-ca`, with context enabled, running just `step-ca` will use the default context.

For example, this will run the ssh context.
```
step context select ssh
step-ca --password-file password.txt
```

Reply:
Sure, will get right on that.